_posts/2024-11-08-this-month-in-servo.md

```diff
@@ -7,487 +7,6 @@ summary:    Plus initial SubtleCrypto support, CSS size keywords, and OpenHarmon
 categories:
 ---
 
-<!--
-- DONE donations 4201/month
-    - DONE 2060/month opencollective
-    - DONE 2141/month github
-    - DONE thanks.dev (10 donors)
-- wpt <https://wpt.servo.org>
-    - 2024-11-04 60% on all of wpt?
-    - 2024-10-31 enabled css-grid tests 34052
-- DONE api features
-    - DONE 2024-09-28 better handling of data: urls 33500
-    - DONE 2024-10-09 `crypto.subtle.generateKey("AES-CBC")` 33628
-    - DONE 2024-10-11 toString() method on DOMMatrix and DOMMatrixReadOnly 33792
-    - DONE 2024-10-12 `DOMRect.fromRect()` 33798
-        - DONE getBounds() method on DOMQuad now handles NaN correctly 33794
-        - DONE is2D attribute on DOMMatrix now updates on mutation 33796
-    - DONE 2024-10-16 fetch metadata headers (‘Sec-Fetch-Site’, ‘Sec-Fetch-Mode’, ‘Sec-Fetch-User’, ‘Sec-Fetch-Dest’) 33830
-    - DONE 2024-10-18 non-ASCII characters in <img srcset> 33873
-    - DONE 2024-10-22 `crypto.subtle.encrypt()` and `decrypt()` for AES-CBC 33795
-    - DONE 2024-10-23 `crypto.subtle.generateKey("AES-CTR")` 33963
-    - DONE 2024-10-25 ‘transition-behavior: allow-discrete’ 33991
-    - DONE 2024-10-27 ‘transition: allow-discrete’ 34005
-    - DONE 2024-10-27 stub interface for IntersectionObserver 33989
-    - DONE 2024-10-30 `crypto.subtle.digest()` 34034
-    - DONE 2024-10-30 ‘mix-blend-mode: plus-lighter’ 34057
-- layout
-    - DONE keyword sizes (‘min-content’, ‘max-content’, ‘fit-content’, ‘stretch’)
-        - DONE prelim 33558
-        - DONE enable stretch 33659
-        - DONE 2024-10-09 floats 33666
-        - DONE 2024-10-11 atomic inlines 33737
-        - DONE 2024-10-23 absolute/fixed positioned elements 33950
-        - DONE 2024-10-23 prevent ‘stretch’ from being negative 33951
-        - DONE 2024-10-27 intrinsic contributions 33854
-    - fixed ‘align-content: <start|end>’ with ‘flex-wrap: wrap-reverse’ 33667
-    - flex container intrinsic size with inline margins 33780
-    - cyclic percentages in ‘min-width’ and ‘min-height’ 33988
-    - video intrinsic size 31746 34069
-    - fixed ‘text-wrap-mode: nowrap’ 33848
-- rendering
-    - correct encoding when setting `href` via the DOM 33825
-- DONE fonts
-    - DONE async FontKey and FontInstanceKey 33600
-    - DONE first font instance creation 33638
-    - DONE memory mapping system fonts on macOS and freetype platforms 33747 mukilan
-- DONE perf
-    - DONE single fetch thread, fixing most errors due to too many file descriptors 33863
-    - unmultiply_inplace 33553 33584 33582
-    - DONE flexbox layout tracing 33647
-    - DONE flexbox caching to avoid unnecessary layouts 33964 33967
-    - DONE avoid recomputing sizes when not using vertical percentages or stretching vertically 33806
-    - DONE avoid recomputing percentage heights of table rows 33575
-    - DONE fixed log spam when not using experimental `--features tracing` 33845
-    - DONE faster builds of script crate 33502
-    - SKIP(reverted) fixed small incremental parser updates 33611
```

Comment:
I don't believe this was actually reverted, and it's a nice performance win to write about.

Reply:
good catch, done in 76bd1163d858dcf12db48f4857676a5bb445f739